### PR TITLE
Tea render cart

### PR DESCRIPTION
**Pull request by TeaChareqishvili**
1. I have created a checkout page to render the user's selected products.
2. I added a folder named "check out buttons" for checkout buttons (delete all, increment , decrement and delete ), and files for each button on the checkout page.
Please check the functionality and approve that it can be merged.

### CODE REVIEW

File: app/[locale]/(dashboard)/checkout/page.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  const cart = await getCart("29");
+  const cartItems = await cart.json();
+  //   console.log(cartItems, "items");
+
+  const productQuantityMap: ProductQuantityMap = [];
+  cartItems.rows.forEach((item: CartItem) => {
+    productQuantityMap[item.productid] = item.quantity;
+  });
+
+  const productPromises = cartItems.rows.map((item: Product) =>
```
Comment (NikolozR) [inline app/[locale]/(dashboard)/checkout/page.tsx]: item should be type of CartItem and the build problem would be resolved

**Review by NikolozR — CHANGES_REQUESTED** [review]

File: app/[locale]/(dashboard)/checkout/page.tsx (hunk 1 above)
Comment (TeaChareqishvili) [reply app/[locale]/(dashboard)/checkout/page.tsx]: okey got it

Comment (TeaChareqishvili) [reply app/[locale]/(dashboard)/checkout/page.tsx]: can u check it now?

**Review by NikolozR — APPROVED** [review]
All good

**Pull request merged**